docs/tutorials/proposals/asset-transfer-proposal.md

```diff
@@ -0,0 +1,195 @@
+
+## Anatomy of an asset transfer proposal
+Read on for the key inputs to a governance-initiated parameter proposal.
+
+**Rationale** requires a title and a description. They are free-text fields that describe the purpose of the proposal. Within the description, include links with more information about your proposal (such as to the IPFS content or forum post) that voters can reference to learn more about the proposal.
+
+The **terms** of the proposal describe the details of the proposal, including closing and enactment timestamps, and details about the proposed transfer, including what accounts the transfer is to move assets between, and optionally information about rewards to fund with the transfer, if that's the intent.
+* **Closing timestamp**, the date and time when voting closes, must be within a range of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.transfer.minClose" hideName={true}/> and <NetworkParameter frontMatter={frontMatter} param="governance.proposal.transfer.maxClose" hideName={true}/>, from when it was proposed.
+* **Enactment timestamp**, the date and time when the transfer is to happen, must be within a range of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.transfer.minEnact" hideName={true}/> and <NetworkParameter frontMatter={frontMatter} param="governance.proposal.transfer.maxClose" hideName={true}/>.
+
+**Amount** is how much to transfer, as whole number with the asset decimal places implied. For example, if the asset has a decimal place of 2 and the transfer is for 100, then the amount needs to be set at 10000. The maximum you can propose to transfer is governance.proposal.transfer.maxAmount. Before proposing, make sure the account you're transferring from exists and has a balance. The full amount may not be transferred if there isn't enough to transfer.
+
+**Fraction of balance** is used to determine what fraction of the source account's balance could be transferred, the biggest fraction that you can choose is <NetworkParameter frontMatter={frontMatter} param="governance.proposal.transfer.maxFraction" hideName={true}/>.
+
+**Governance transfer type** lets you determine if you want to try to transfer all or nothing, or to transfer as much as is possible up to the full amount.
+
+### Fields used to fund trading rewards
+
+If you're proposing a transfer to fund rewards, there are extra fields to define.
+
+The `toAccountType` must be the account type that matches the reward category. For example, to propose that the 'average position' reward will pay out, you'll need to set the "to account type" type as `ACCOUNT_TYPE_REWARD_AVERAGE_POSITION`, and then choose the complementary reward category, known as the `metric`. The asset you choose then determines which market(s) the reward targets.
+
+The `to` field is used for public keys. In the case of network-managed accounts, the input they need is `0000000000000000000000000000000000000000000000000000000000000000`.
```

Comment:
there's no `to` field, there's `destination` field and the above is also inaccurate. The source [destination] is the public key of the party when the source [destination] account type is a party account. Otherwise, it depends on the context - if we're transferring to/from market's insurance pool, then the source [destination] is the market ID.